src/test/java/org/jenkinsci/plugins/kubernetes/credentials/OpenShiftBearerTokenCredentialTest.java

```diff
@@ -33,37 +32,36 @@ public class OpenShiftBearerTokenCredentialTest {
     @Rule
     public ExpectedException expectedEx = ExpectedException.none();
 
-    private Server server;
+    private HttpServer server;
+    private String mockserverBaseUrl;
 
     @Before
     public void prepareFakeOAuthServer() throws Exception {
-        InetSocketAddress addr = new InetSocketAddress("localhost", 0);
-        server = new Server(addr);
-        ServletContextHandler context = new ServletContextHandler();
-        context.setContextPath("/");
-        context.addServlet(new ServletHolder(new MockHttpServlet()), "/*");
-        server.setHandler(context);
+        server = HttpServer.create(new InetSocketAddress("localhost", 0), 0);
+        OpenShiftBearerTokenCredentialMockServer.registerHttpHandlers(server);
         server.start();
+        mockserverBaseUrl = "http:/"+server.getAddress()+"/";
```

Comment:
Does that result in a valid URL?  Is the first character of the `server.getAddress()` value a '/'?  If not, then won't that baseUl be 'http:/address' when it should be 'http://address'?
```suggestion
        mockserverBaseUrl = "http:/"+server.getAddress()+"/";
```

Reply:
I confirmed that it generates a valid URL.  Question answered.  The code is correct.

Reply:
> I confirmed that it generates a valid URL. Question answered. The code is correct.

Yes, the code is correct, but it is illegible.